src/args.rs

```diff
@@ -0,0 +1,37 @@
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct PokerArgs {
+    #[command(subcommand)]
+    /// Draw hands or get statistics
+    pub command: Option<ArgsCommands>,
```

Comment:
If we make this:
```suggestion
    pub command: ArgsCommands,
```
Then I think that we could provide a default value for `command` (eg. print sorted deck, or whatever), and it would let us remove the special case check at the end of `main`.

Reply:
I can't find something to do with this. I tried what you suggested but I couldn't find any command to get a default value. We can't use command for that but I use it as arg then I can do it. I have two option either to let it remain as it is with enum structure or I can change it to arguments

Reply:
Ah - ok.  Thanks for explaining. I did a bit of reading to understand this better (https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_value), and it seems (like you pointed out), that the command arguments can have default values, but not the commands themselves.

Let's leave it as enum arguments (ie. top-level commands are "draw hand", "statistics", and "display deck"), and then use regular rust code to manage the case where the use doesn't specify anything.